spinalcordtoolbox/deepseg/models.py

```diff
@@ -100,7 +108,10 @@
          'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel']},
     'seg_exvivo_gm-wm_t2':
         {'description': 'Grey/white matter seg on exvivo human T2w.',
-         'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']}
+         'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']},
+     'seg_gm_sc_7t_t2star':
```

Comment:
Our "Linting" checker has noticed some style issues with line 112 and line 114: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3527/checks?check_run_id=3758687437

Here is some more information about the specific style issues:

* https://www.flake8rules.com/rules/E131.html
* https://www.flake8rules.com/rules/W291.html

Could you please update the code to fix these issues? Thank you!